_docker-compose.override.yml

```diff
@@ -0,0 +1,14 @@
+# use this file to start the db service with example data 
+# provided by the nisq and planqk content repositories
+# make sure the ./planqk-atlas-content-ssh_secret file is present 
```

Comment:
typo: should be ssh_secret as you renamed it?

Reply:
fixed